guides/common/modules/proc_obtaining-the-location-of-the-ca-bundle-for-project.adoc

```diff
@@ -0,0 +1,27 @@
+[id="obtaining-the-location-of-the-ca-bundle-for-{project-context}"]
+= Obtaining the location of the CA bundle for {Project}
+
+[role="_abstract"]
+To deploy the MCP server for {Project}, the CA bundle for {Project} must be available on the system where the MCP server is being deployed.
+The CA bundle is the certificate of a CA which signed the certificate that is used to secure {ProjectWebUI}.
+
+Perform these steps on your {ProjectServer} or ask your administrator to perform them for you.
```

Comment:
From the writing perspective, this was a bit tricky because while we want Managing hosts to serve the user persona, this particular part of the MCP server setup needs an admin. I solved this by introducing a new module for this de facto prerequisite and adding this line that explicitly mentions the admin persona.